models/CurrentState.php

```diff
@@ -67,15 +72,9 @@ class CurrentState extends State
      */
     public function cast()
     {
-        if ($this->transitions !== null && !$this->transitions instanceof EntitySet) {
```

Comment:
Why is this cast removed?

Reply:
Because the same cast is performed in parent State class